addons/web/static/tests/views/fields/many2many_field_tests.js

```diff
@@ -1821,4 +1823,85 @@ QUnit.module("Fields", (hooks) => {
         await click(target.querySelector(".oe_kanban_global_click"));
         assert.verifySteps(["doActionButton type object name a1"]);
     });
+
+    QUnit.test("many2many active_id in field context -- in list", async (assert) => {
+        assert.expect(3);
+        serverData.models.partner_type.fields.partner_id = {
+            string: "Partners",
+            type: "many2one",
+            relation: "partner",
+        };
+        serverData.views = {
+            "partner_type,false,form": `<form><field name="partner_id" /></form>`,
+        };
+
+        const nameSearch = makeDeferred();
+        await makeView({
+            type: "list",
+            resModel: "partner",
+            serverData,
+            arch: `<tree editable="top"><field name="timmy" widget="many2many_tags" context="{ 'default_partner_id': active_id}"/></tree>`,
+            mockRPC(route, args) {
+                if (args.method === "onchange") {
+                    assert.strictEqual(args.kwargs.context.default_partner_id, 1);
+                }
+                if (args.method === "name_search") {
+                    nameSearch.resolve();
+                }
+            },
+        });
+
+        await click(target.querySelector(".o_data_cell"));
+        await editInput(target, ".o_field_many2many_selection input", "indianapolis");
+        await nextTick();
+        await nameSearch;
+        await nextTick();
```

Comment:
why do you need this

Reply:
otherwise the dropdown is not present in the dom